apps/web/public/static/locales/en/common.json

```diff
@@ -1236,6 +1236,8 @@
   "may_require_confirmation": "May require confirmation",
   "nr_event_type_one": "{{count}} event type",
   "nr_event_type_other": "{{count}} event types",
+  "nr_team_one": "{{count}} team",
+  "nr_team_other": "{{count}} teams",
```

Comment:
I think we could use a more descriptive name, I really don't know what `nr` means here.